network/peer/peer.go

```diff
@@ -774,17 +781,67 @@ func (p *peer) handle(msg message.InboundMessage) {
 }
 
 func (p *peer) handlePing(msg *p2p.Ping) {
-	p.observeUptimes(msg.Uptime, msg.SubnetUptimes)
```

Comment:
Previously this function could have started closing the connection, but then we would have still attempted to send a pong message... Which is pretty odd... We no longer handle the uptimes in `Pong` messages, so this function was just inlined.